src/foam/nanos/notification/NotificationExpansionDAO.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.notification',
+  name: 'NotificationExpansionDAO',
+  extends: 'foam.dao.ProxyDAO',
+
+  documentation: `Expands a notification from a group or broadcast notification to a user specific notification. `,
```

Comment:
Doesn't SendNotificationDAO already do this?

Reply:
SendNotificationDAO and all other Notification DAO decorators have been removed.